tests/integration/test_sca/test_enabled.py

```diff
@@ -0,0 +1,139 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: These tests will check if the `enabled` option of the SCA module
+       is working correctly. This option is located in its corresponding section of
+       the `ossec.conf` file and allows enabling or disabling this module.
+
+components:
+    - sca
+
+targets:
+    - manager
+    - agent
+daemons:
+    - wazuh-modulesd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/index.html
+
+tags:
+    - sca
+'''
+import os
+import pytest
+
+from wazuh_testing import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.modules.sca import event_monitor as evm
+from wazuh_testing.modules.sca import SCA_DEFAULT_LOCAL_INTERNAL_OPTIONS as local_internal_options
+
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0)]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_enabled.yaml')
+cases_path = os.path.join(TEST_CASES_PATH, 'cases_enabled.yaml')
+
+# Test configurations
+configuration_parameters, configuration_metadata, case_ids = get_test_cases_data(cases_path)
+configurations = load_configuration_template(configurations_path, configuration_parameters, configuration_metadata)
+
+
+
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=case_ids)
+def test_enabled(configuration, metadata, prepare_cis_policies_file, truncate_monitored_files, set_wazuh_configuration,
+                 configure_local_internal_options_function, restart_wazuh_function):
+    '''
+    description: Check that sca is started when is set enabled yes. When enabled is set to no, the test will
+                 check that the sca is disabled and does not start.
```

Comment:
It could be a more general description in this case. Something like checking the behavior of sca when it is enabled and disabled.

Reply:
done in https://github.com/wazuh/wazuh-qa/pull/3566/commits/de6653c714466ea5b9e6a6e63fddee95d86ca4d5